overlord/snapstate/storehelpers_test.go

```diff
@@ -483,3 +485,74 @@ func (s *snapmgrTestSuite) TestInstallSizeRemotePrereq(c *C) {
 	op := s.fakeStore.fakeBackend.ops.MustFindOp(c, "storesvc-snap-action:action")
 	c.Assert(op.action.InstanceName, Equals, "snap-content-slot")
 }
+
+func (s *snapmgrTestSuite) TestSnapHoldsSnapsOnly(c *C) {
+	st := s.state
+	st.Lock()
+	defer st.Unlock()
+
+	mockInstalledSnap(c, st, snapAyaml, false)
+	mockInstalledSnap(c, st, snapByaml, false)
+	mockInstalledSnap(c, st, snapCyaml, false)
+
+	//mockLastRefreshed(c, st, "2021-05-09T10:00:00Z", "snap-a", "snap-b", "snap-c")
+
+	//now, err := time.Parse(time.RFC3339, "2021-05-10T10:00:00Z")
+	//c.Assert(err, IsNil)
+	//restore := snapstate.MockTimeNow(func() time.Time {
+	//	return now
+	//})
+	//defer restore()
```

Comment:
I don't remember exactly but I think this might be a left over from when HoldRefresh worked differently? I think it can be removed.